lms/djangoapps/verify_student/tests/test_services.py

```diff
@@ -0,0 +1,64 @@
+# encoding: utf-8
+"""
+Tests of reverify service.
+"""
+import ddt
+from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
+from xmodule.modulestore.tests.factories import CourseFactory, ItemFactory
+from student.tests.factories import UserFactory, CourseEnrollmentFactory
```

Comment:
`CourseEnrollmentFactory` is unused, which is probably causing your build to fail quality checks.